.github/workflows/dev-deploy-widget.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     uses: ./.github/workflows/reusable-widget-deploy.yml
     with:
       environment: Development
-      react_app_api_url: https://dev.api.novu.co
+      react_app_api_url: https://api.novu-staging.co
```

Comment:
If we remove this line from dev deployments, we can make both `novu-staging` and `dev.dashboard.novu.co` work in parallel for the next couple of days that we need to test the migration.

Reply:
this is in widget, did you mean in the config above for web?

Reply:
🤦 yes!